openfe/protocols/openmm_utils/omm_settings.py

```diff
@@ -134,62 +134,72 @@ class Config:
     or `independent` (independently sampled lambda windows).
     Default `repex`.
     """
-    online_analysis_interval: Optional[int] = 250
+    steps_per_iteration = 250 * unit.timestep  # todo: IntQuantity
     """
-    MCMC steps (i.e. ``IntegratorSettings.n_steps``) interval at which
+    Number of integration timesteps between each time the MCMC move
+    is applied. Default 250 * unit.timestep.
+    """
+    real_time_analysis_interval: Optional[int] = 250
```

Comment:
This isn't `unit.timestep` as it's a multiple of steps_per_iteration right?  So the default is 250 * 250?

Reply:
I think this is answered in the docstring below >.<